test/StakeManager.js

```diff
@@ -1204,5 +1205,43 @@ describe('StakeManager', function () {
       const tx = stakeManager.connect(signers[3]).withdraw(epoch, stakerIdacc3);
       await assertRevert(tx, 'Nonpositive Stake');
     });
+
+    it('staker should not be able to withdraw more than his sToken balance', async function () {
```

Comment:
Please rename this to 
Delegators/Stakers should not be able to withdraw if their current balance is less than the locked amount.

Reply:
> Please rename this to
> Delegators/Stakers should not be able to withdraw if their current sRZR balance is less than the locked amount.

I am only checking for staker here so can I rename to "Stakers should not be able to withdraw if their current sRZR balance is less than the locked amount"  @abhishekvispute

Reply:
yes